react-form-controls/my-react-form-controls/src/RegistrationFormControlled.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+
+    console.log('username:', username, 'password:', password);
+  }
+
+  return (
+    <form method="post" onSubmit={handleSubmit}>
```

Comment:
Since we aren't sending the form to the backend, the `method` prop is not necessary.